### ISSUE TITLE
[Accessibility] Add missing table headers

### ISSUE THREAD

**Issue by gewarren** [dotnet/dotnet-api-docs#9813]
https://github.com/search?q=repo%3Adotnet%2Fdotnet-api-docs+%2F%5C%7C%5C%7C%5C%7C%2F+language%3AXML&type=code&l=XML

---
[Associated WorkItem - 356819](https://dev.azure.com/msft-skilling/Content/_workitems/edit/356819)
